pkg/cmd/pr/checks/checks.go

```diff
@@ -65,208 +78,77 @@ func NewCmdChecks(f *cmdutil.Factory, runF func(*ChecksOptions) error) *cobra.Co
 	}
 
 	cmd.Flags().BoolVarP(&opts.WebMode, "web", "w", false, "Open the web browser to show details about checks")
+	cmd.Flags().BoolVarP(&opts.Watch, "watch", "", false, "Watch checks until they finish")
+	cmd.Flags().IntVarP(&interval, "interval", "i", defaultInterval, "Refresh interval in seconds when using `--watch` flag")
```

Comment:
Cobra has a duration flag which I thought about using here but was unsure since we went with an int flag for `run watch`.

Reply:
My bad, I didn't know about the flag type. Nice catch! :+1: